hal/nxp_p1021.c

```diff
@@ -420,6 +423,18 @@ enum elbc_amask_sizes {
 #define ESPI_CSMODE_CSAFT(x) (((x) & 0xF) << 8)  /* CS assertion time in bits after frame end */
 #define ESPI_CSMODE_CSCG(x)  (((x) & 0xF) << 3)  /* Clock gaps between transmitted frames according to this size */
 
+#if defined(ENABLE_ESPI) || defined(ENABLE_DDR)
+static void udelay(uint32_t delay_us)
+{
+    uint32_t i;
+    static const uint32_t oneus = (400000000 / 1000000);
```

Comment:
I think we should use a macro for clock speed of the chip to do this calculation

Reply:
Will fix. It is a macro.